src/browser/WindowControls.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+
+  'use strict';
+
+  const Component = require('omniscient');
+  const {DOM} = require('react')
+  const {sendEventToChrome} = require('./actions');
+
+  const styleContainer = {
+    alignItems: 'center',
+    flexShrink: 0,
+    marginLeft: 7
+  };
+
+  const styleButton = {
+    width: 12,
+    height: 12,
+    marginRight: 8,
+    borderRadius: '50%'
+  };
+
+  const styleGreyButton = Object.assign({
+    backgroundColor: 'hsl(0, 0%, 86%)'
+  }, styleButton);
+
+  const WindowControls = Component(({isDocumentFocused, windowControls, theme}) => {
+
+    const styleMinButton = Object.assign({}, theme.windowMinButton, styleButton);
+    const styleMaxButton = Object.assign({}, theme.windowMaxButton, styleButton);
+    const styleCloseButton = Object.assign({}, theme.windowCloseButton, styleButton);
```

Comment:
I think it would be useful add an utility function to our toolchain similar to `classSet` for mixing styles, maybe something along these lines:

```js
const isObject = x => x && typeof(x) === "object"
const mix = (...styles) => Object.assign({}, ...styles.filter(isObject))

const styleCloseButton = mix(theme.windowCloseButton, styleButton)
```